buckw

```diff
@@ -75,20 +75,26 @@ getChanges ( ) {
             ["suffix", "gradle"],
             ["name", ["gradle-wrapper.properties", "lint.xml"]],
             ["match", "**/src/**/AndroidManifest.xml", "wholename"]
+        ],
+        ["not",
+            ["dirname", ".okbuck"]
         ]
     ],
     "fields": ["name"]
 }]
 EOT`
     SOURCE_ROOTS=`watchman --output-encoding=json --no-pretty -j 2>&1 <<-EOT
-["query", ".", {
+["query", "$WORKING_DIR", {
```

Comment:
I think you would want to have the `$WORKING_DIR` as the root dir as excludes and includes will be added relative to that.